src/pc/1.15/ChunkColumn.js

```diff
@@ -190,10 +195,6 @@ module.exports = (Block, mcData) => {
       this.biomes[getBiomeIndex(pos)] = biome
     }
 
-    setBiomeColor (pos, r, g, b) {
```

Comment:
this needs to stay in existence due to the fact it exists in some mcpe (unless you remove these versions completely)

Reply:
They still exist, they've just been moved to `CommonChunkColumn` which this extends

Reply:
only the pc/* versions have the getBiomeColor/setBiomeColor moved to pc/common/CommonChunkColumn, so mcpe versions shouldn't be effected